tests/foreman/cli/test_repository.py

```diff
@@ -691,3 +699,162 @@ def test_positive_upload_content(self):
             "Successfully uploaded file '{0}'".format(RPM_TO_UPLOAD),
             result[0]['message'],
         )
+
+
+class RepositoryExportTestCase(CLITestCase):
+    """Tests for exporting a repository via CLI"""
+
+    def setUp(self):
```

Comment:
Actually, it should be enough to run this setup once for all tests. The thing is, by using setUpClass we'll loose the ability to use unittest assertions, which will make debugging much harder later. Or I could use built-in assertions here, but i remember @elyezer once told me why we're avoiding them, some compatibility issues or smth... @elyezer could you please remind me?